release_docs/RELEASE.txt

```diff
@@ -89,6 +89,13 @@ New Features
 
     Java Library:
     -------------
+    - Added version of H5Rget_name to return the name string.
+
+      Other functions that get_name process the get_size then get the name
+      within the JNI implementation. Now H5Rget_name has a H5Rget_name_string.
```

Comment:
I can't make sense of the first sentence.  Maybe words missing?   Can you make it clearer?